integration/airflow/openlineage/airflow/extractors/base.py

```diff
@@ -67,13 +64,11 @@ def patch(self):
         pass
 
     @classmethod
-    def get_operator_class(cls):
-        return cls.operator_class
+    def get_operator_classnames(cls) -> List[str]:
```

Comment:
_Minor_: Mind if we add docs to this method?

Reply:
Added doc about this:

In Airflow 2 some operators are deprecated and simply subclass the new 
implementation, for example BigQueryOperator: 
https://github.com/apache/airflow/blob/main/airflow/contrib/operators/bigquery_operator.py
The BigQueryExtractor needs to work with both of them.